Connector/btc/apirpc.py

```diff
@@ -21,11 +22,17 @@ def getAddressHistory(id, params, config):
     if err is not None:
         raise error.RpcBadRequestError(id=id, message=err.message)
 
-    addrHistory = RPCConnector.request(
-        endpoint=config.electrumRpcEndpoint,
+    try:
+        scriptHash = utils.scriptHash.addressToScriptHash(params["address"])
+    except Exception:
+        raise error.RpcBadRequestError(id=id, message=INVALID_ADDRESS_ERROR)
+
+    addrHistory = RPCSocketConnector.request(
+        hostname=config.electrsEndpoint.split(":")[0],
```

Comment:
This can't work properly since there are no changes in the file `/Connector/btc/config.py`. This file contains all the configuration of the endpoints.

Because of the deletion of  `electrsEndpoint` variable in other files, this PR will never work with the `electrsHost` and `electrsPort` variables.

There are two ways to fix this:
1. **Best option**: Just keep the `electrsEndpoint` variable in `availableCurrencies.json` and `defaultConfig.json`. So you can combine `electrsHost` and `electrsPort` in the old variable.

2. **Not so good option**:  You can modify the `btc/config.py` and add this `electrsHost`/`electrsPort` getter and setter to use them in other files.